apis/facebook/api/src/objects/fb_group.e

```diff
@@ -0,0 +1,85 @@
+note
+	description: "[
+			Represents a Facebook group.
+	]"
+	author: ""
+	date: "$Date$"
+	revision: "$Revision$"
+	EIS: "name= Facebook Group", "src=https://developers.facebook.com/docs/graph-api/reference/v2.9/group", "protocol=uri"
+class
+	FB_GROUP
+
+inherit
+
+	DEBUG_OUTPUT
+
+feature -- Access
+
+	id: detachable STRING
```

Comment:
is the id ... a number as string, a ascii string? a utf_8 string? a unicode string (in this case should be STRING_32) ?